compiler/luci-interpreter/src/loader/KernelBuilder.cpp

```diff
@@ -48,7 +49,7 @@ class KernelBuilderRegistry
   using KernelBuilderFunc = std::unique_ptr<Kernel>(const luci::CircleNode *,
                                                     KernelBuilderHelper &);
 
-  KernelBuilderRegistry()
+  KernelBuilderRegistry() : _operator_builders(size_t(BuilderId::size), nullptr)
```

Comment:
In accordance with https://github.com/Samsung/ONE/pull/7699/files#r710018921:
```suggestion
  KernelBuilderRegistry() : _operator_builders(size_t(BuilderId::Size), nullptr)
```